### PR TITLE
1127: Updated oidc internal documentation

### PR DESCRIPTION
**Pull request by tuj**
#### Link to ticket

https://leantime.itkdev.dk/#/tickets/showTicket/1127

#### Description

Updated oidc internal documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,9 +128,16 @@ The internal oidc provider gets that user's name, email and tenants from claims.
 
 The claim keys needed are set in the env variables:
 
-- INTERNAL_OIDC_CLAIM_NAME
-- INTERNAL_OIDC_CLAIM_EMAIL
-- INTERNAL_OIDC_CLAIM_GROUPS
+- `INTERNAL_OIDC_CLAIM_NAME`
+- `INTERNAL_OIDC_CLAIM_EMAIL`
+- `INTERNAL_OIDC_CLAIM_GROUPS`
```
Comment (rimi-itk) [inline README.md:133]: Why are these prefixes with `INTERNAL_`? Isn't `OIDC_CLAIM_NAME`, say, be sufficient.

File: README.md [hunk 2]
```diff
@@ -128,9 +128,16 @@ The internal oidc provider gets that user's name, email and tenants from claims.
...
+
+The value of the claim with the name that is defined in the env variable `INTERNAL_OIDC_CLAIM_GROUPS` is mapped to
+the user's access to tenants in `App\Security\AzureOidcAuthenticator`. The claim field should consist of an array of
+names that should follow the following structure `<TENANT_NAME><ROLE_IN_TENANT>`.
```
Comment (rimi-itk) [inline README.md:137]: Having a non-alphanumeric separator between `<TENANT_NAME>` and `<ROLE_IN_TENANT>` would probably be a good idea, e.g. `<TENANT_NAME>\<ROLE_IN_TENANT>`.

**Review by rimi-itk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tuj) [reply README.md:133]: We have two types of OIDC login (Internal/External)

File: README.md (hunk 2 above)
Comment (tuj) [reply README.md:137]: Agreed. But that is not how it has been implemented.
The values for <ROLE_IN_TENANT> should be configurable:
https://github.com/os2display/display-api-service/blob/develop/src/Security/AzureOidcAuthenticator.php#L34

**Pull request merged**